src/components/events/EventCard.tsx

```diff
@@ -1,7 +1,28 @@
-const EventCard = () => {
+import React from "react";
+import Image from "next/image";
+
+interface EventCardProps {
+  image: string;
+  text: string;
+  title: string;
+}
+const EventCard: React.FC<EventCardProps> = ({ image, text, title }) => {
   return (
-    <div className="">
-      <p>EventCard</p>
+    <div className="border-3 mx-auto flex h-80 w-64 flex-row items-center justify-center rounded-3xl border-white bg-white shadow-2xl">
```

Comment:
flex-row is unnecessary here, also try to get height and width to be percentages or use vw so they adjust based on screen size